app/views/manage_proposal/show.html.erb

```diff
@@ -1,4 +1,21 @@
 <div class="row row-content manage-proposals-mmt">
+  <% if @ingest_errors && !@ingest_errors.empty? %>
+    <section class="errors">
+      <div class="eui-banner--danger">
+        <div class="eui-banner__message">
+          <h4><i class="fa fa-exclamation-triangle"></i> This proposal has the following errors:</h4>
+          <ul class="no-bullet">
+            <% @ingest_errors.each do |error| %>
```

Comment:
Haven't we seen examples of this where the errors are more then one screen's worth? Going forward what if we limited to the first 20 or so errors?

Reply:
I"m thinking this should be a future ticket